videos/README.md

```
@@ -0,0 +1,51 @@
+# Videos — en-US
+
+This directory contains video script files for the English language.
+
+## Directory Structure
+
+Each supported language has its own subdirectory following the pattern:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)